README.md

```diff
@@ -4,3 +4,5 @@ Test repo for pull requests
 Adding a line at github/jbullock35gmail.
 
 Adding a second line at github/jbullock35gmail. [748am]
+
+Adding a line from local repo when in the myNewBranch branch.  [812am]
```

Comment:
From jbullock35gmail, leaving comment on another range of lines. Choosing "Start a review."